internal/agents/fuzzy_matcher.go

```
@@ -694,43 +704,43 @@ func GetCategorizedAmplifiers(allowFactoryCaptures bool) string {
+				continue
+			}
+			if seen[a.Name] {
+				continue
+			}
+			seen[a.Name] = true
+			names = append(names, a.Name)
+		}
+		if len(names) == 0 {
+			continue
+		}
+		sb.WriteString(fmt.Sprintf("\n%s:\n", strings.ToUpper(archetype)))
```

golangci-lint (2.12.2)

[error] QF1012: Use fmt.Fprintf(...) instead of WriteString(fmt.Sprintf(...))

(staticcheck)